doc/nrf/gs_testing.rst

```diff
@@ -21,3 +21,26 @@ Connect with the following settings:
  * 1 stop bit
  * No parity
  * HW flow control: None
+
+.. _lte_connect:
+
+How to connect with LTE Link Monitor
+************************************
+
+To connect to nRF9160 based board devices, you can also use `LTE Link Monitor`_ which is implemented in `nRF Connect for Desktop`_.
```

Comment:
```suggestion
To connect to nRF9160-based boards (for example, the nRF9160 DK or Nordic Thingy:91), you can also use `LTE Link Monitor`_, which is implemented in `nRF Connect for Desktop`_.
```
"board devices" sounds a bit strange. Let's just call them boards and give Thingy as an example. Or we could use device instead of board, up to you. :)

"which" needs a comma - see https://www.quickanddirtytips.com/education/grammar/which-versus-that